ecommerce/forms.py

```diff
@@ -146,3 +153,166 @@ class LoginForm(FlaskForm):
     password = PasswordField('Password', validators=[DataRequired()])
     remember = BooleanField('Remember Me')
     submit = SubmitField('Login')
+
+#START CART MODULE
+#Gets products in the cart-Akanksha
```

Comment:
@akankshanegarora - can u remove name

Reply:
Why? Any issues with that?

Reply:
No one putting any names on routes or forms. We should the same standard through the project.

Reply:
I just added on functions I implemented in forms just in case they have any problems, just to know. I was just following company practices. I would remove if you want on next commit.